yaksh/templates/yaksh/monitor.html

```diff
@@ -80,10 +88,11 @@
         <td> {{ paper.user.username }} </td>
         <td> {{ paper.user.profile.roll_number }} </td>
         <td> {{ paper.user.profile.institute }} </td>
-        <td> {{ paper.get_answered_str }} </td> 
+        <td> {{ paper.get_answered_str }} </td>
         <td> {{ paper.marks_obtained }} </td>
         <td> {{ paper.answers.count }} </td>
-        <td> {{ paper.status }} </td>
+        <td id="time_left{{forloop.counter0}}"> {{ paper.time_left }} </td>
```

Comment:
Why are you doing this with JS, can't this be done server side and just sent to the template to render out?